source/sourcing_and_processing_data/processing.rst

```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
     * Field
     * yes
 
+what3words
+^^^^^^^^^^
+
+Get 3 word address from coordinates. Requires a what3words API key to be configured by a domain administrator.
+
+It takes the following parameters:
+
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * Language
+    * String containing the desired language for the 3 word address. Default is English
+    * String
+    * No
+  * * Coordinates
+    * Field containing the geographical coordinates to geocode
+    * Field
+    * yes
+  * * Output field
+    * Field that will contain the resulting 3 word address
+    * Field
+    * yes
+
+Get coordinates from a 3 word address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert a 3 word address into coordinates. Requires a what3words API key to be configured by a domain administrator.
```

Comment:
"**This processor** convert**s** a 3 word address into **geographical** coordinates. **A what3words API key is required for this processor.**"